CondTools/BeamSpot/test/BeamSpotOnlineRecordsWriter_cfg.py

```diff
@@ -10,7 +10,7 @@
                  VarParsing.VarParsing.varType.bool, # string, int, or float
                  "are we running the unit test?")
 options.register('inputFile',
-                 "BeamFitResults_Run306171.txt", # default value
+                 "../data/BeamFitResults_Run306171.txt", # default value
```

Comment:
I think this is the change that makes the unit test to fail.

Reply:
I agree, I will fix this asap.

Reply:
done in e5a659f